### PR TITLE
adjust order of side bars

### PR DESCRIPTION
**Pull request by sonichi**
## Why are these changes needed?

Adjust the order of side bars to make it easier to follow.
Improve documentation.
Prepare for new release v0.2.20.

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by gagb — APPROVED** [review]

File: autogen/agentchat/chat.py [hunk 1]
```diff
@@ -138,19 +138,19 @@ def initiate_chats(chat_queue: List[Dict[str, Any]]) -> List[ChatResult]:
         chat_queue (List[Dict]): a list of dictionaries containing the information about the chats.
 
         Each dictionary should contain the input arguments for `ConversableAgent.initiate_chat`. For example:
```
Comment (qingyun-wu) [inline autogen/agentchat/chat.py:140]: Add a link to ConversableAgent.initiate_chat?

File: autogen/agentchat/conversable_agent.py [hunk 2]
```diff
@@ -880,66 +880,72 @@ def initiate_chat(
...
+            ):
+                return recipient.last_message(sender)["content"]
+            ```
             summary_args (dict): a dictionary of arguments to be passed to the summary_method.
-                    E.g., a string of text used to prompt a LLM-based agent (the sender or receiver agent) to reflext
-                    on the conversation and extract a summary when summary_method is "reflection_with_llm".
-                    Default is DEFAULT_SUMMARY_PROMPT, i.e., "Summarize takeaway from the conversation. Do not add any introductory phrases. If the intended request is NOT properly addressed, please point it out."
+                One example key is "summary_prompt", and value is a string of text used to prompt a LLM-based agent (the sender or receiver agent) to reflect
+                on the conversation and extract a summary when summary_method is "reflection_with_llm".
+                Default is DEFAULT_SUMMARY_PROMPT, i.e., "Summarize takeaway from the conversation. Do not add any introductory phrases. If the intended request is NOT properly addressed, please point it out."
```
Comment (qingyun-wu) [inline autogen/agentchat/conversable_agent.py:905]:
```suggestion
                The default summary_prompt is DEFAULT_SUMMARY_PROMPT, i.e., "Summarize takeaway from the conversation. Do not add any introductory phrases. If the intended request is NOT properly addressed, please point it out."
```

File: autogen/agentchat/chat.py (hunk 1 above)
Comment (sonichi) [reply autogen/agentchat/chat.py:140]: yes

**Comment by gagb** [comment]
@sonichi https://github.com/microsoft/autogen/pull/2040/files#diff-c2cfa3b1948c53e5e734bd126473aa55c2d09810cc7075bb488963f9639a0aa1 overwrite the formatting fix from #2041. Was this intentional? Sorry for missing this in my review.

**Pull request merged**